test/core/login/login.test.ts

```diff
@@ -523,6 +523,50 @@ describe('duress', function () {
     expect(topicAccount.isDuressAccount).equals(true)
   })
 
+  it('Avoid creating duress account when using loginWithPassword', async function () {
+    const world = await makeFakeEdgeWorld([fakeUser], quiet)
+    const context = await world.makeEdgeContext(contextOptions)
+    const otherAccount = await context.createAccount({
+      username: 'new-user',
+      pin: '1111'
+    })
+    // Create duress account:
+    await otherAccount.changePin({ pin: '0000', forDuressAccount: true })
+    await otherAccount.logout()
+    // Activate duress mode:
+    const duressAccount = await context.loginWithPIN('new-user', '0000')
+    await duressAccount.logout()
+
+    // Login to the main account using password:
+    const topicAccount = await context.loginWithPassword(
+      fakeUser.username,
+      fakeUser.password
+    )
```

Comment:
Also test login with key?

Reply:
I did this in the first commit: https://github.com/EdgeApp/edge-core-js/pull/654/commits/df9e25b6e36ae1d3a872907d0573a754da11ba80#diff-1fe6c58a4991bd093efe5acca148ea511af9af714e0e17ff1ae54e9865f45fccR550

The second commit added the other test cases with other login methods.